SitumWayfinding/Classes/WayfindingLibrary/SitumMap.swift

```diff
@@ -70,17 +70,29 @@ public protocol SitumMap {
     func setOnMapReadyListener(listener: OnMapReadyListener?)
 
     /**
-     Select a poi in the map
+     Select a poi in the map centering view in the poi and showing the label (if any) on top of poi icon
```

Comment:
Realmente o que se fai é:
* Deseleccionar outro POI se o houbese
* Cambiar de piso se é necesario
* Centrar a vista no Poi
* Actualizar o texto da barra de estado 
* Amosar o nome encima do poi
* Amosar o boton de navegar a él.
Como é moi longo e pode cambiar, que che parece algo como -> Select a given poi. This method will perform the proper actions over the User Interface to make that Poi the selected one?